### PR TITLE
Add eliuds-eggs

### PR DESCRIPTION
**Pull request by BNAndras**
For #48in24. Forum thread at https://forum.exercism.org/t/add-additional-exercises-exercises/10727/3

### CODE REVIEW

**Comment by BNAndras** [comment]
The files worked locally on 26. I’ll troubleshoot it this weekend.

**Comment by NobbZ** [comment]
I sadly won't have the time to help any debugging before tomorrow, but be assured, I never have seen that error before.

**Comment by BNAndras** [comment]
CI passes and I've got a better grasp on the tooling. :)

File: exercises/practice/eliuds-eggs/.meta/example.erl [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-module(example).
+
+-export([eggCount/1]).
+
+
+eggCount(Number) -> eggCount(Number, 0).
+
+eggCount(0, Acc) -> Acc;
+eggCount(Number, Acc) when Number > 0 ->
+    eggCount(Number bsr 1, Acc + (Number band 1)).
```
Comment (NobbZ) [inline exercises/practice/eliuds-eggs/.meta/example.erl:10]:
```suggestion
-export([egg_count/1]).


egg_count(Number) -> egg_count(Number, 0).

egg_count(0, Acc) -> Acc;
egg_count(Number, Acc) when Number > 0 ->
    egg_count(Number bsr 1, Acc + (Number band 1)).
```

Please stay consistent with the naming conventions in the repo. Unless I missed some exercises, snake case is preferred for function names.

**Review by NobbZ — CHANGES_REQUESTED** [review]

**Review by NobbZ — APPROVED** [review]

**Pull request merged**